packages/turbo/ios/RNVisitableView.swift

```diff
@@ -34,72 +65,64 @@ class RNVisitableView: UIView, SessionSubscriber {
   required init?(coder aDecoder: NSCoder) {
     super.init(coder: aDecoder)
   }
-  
-  public func getRNSesssion() -> RNSession? {
-    if (!Thread.isMainThread) {
-      print("getSession accessed from incorrect thread")
-      return nil
-    }
     
-    guard let sessionModule = self.bridge?.uiManager.moduleRegistry.module(forName: "RNVisitableViewModule") as? RNVisitableViewModule else {
-        print("Couldn't find session for sessionHandle:", sessionHandle ?? "default session")
-        return nil
-    }
-    return sessionModule.getSession(sessionHandle: sessionHandle)
-  }
-  
-  public func getSession() -> Session? {
-    return getRNSesssion()?.turboSession
+  public func getWebViewConfiguration() -> WKWebViewConfiguration{
```

Comment:
We've changed most of the other accessor methods to `lazy var` - is there a reason why we haven't done this here?

Reply:
No reason, we can use `lazy` here